### PR TITLE
Fix #2295

### LINKED ISSUE #2295: lint-staged:es fail to lint .eslintrc.js
### Version
2.1.4

### Environment
none

### Reproduction link
[https://github.com/ant-design/ant-design-mobile/](https://github.com/ant-design/ant-design-mobile/)

### Steps to reproduce
Run `npm run lint-staged:es`

### What is expected?
Lint all *.js files without warnings

### What is actually happening?
Get a warning

> warning  File ignored by default.  Use a negated ignore pattern (like "--ignore-pattern '!<relative/path/to/filename>'") to override

and eslint will ignore .eslintrc.js

---
重写链接是什么鬼？这是在本机执行发现的啊~

### PR DESCRIPTION
**Pull request by arniu**
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

---
This change is [Reviewable](https://reviewable.io/reviews/ant-design/ant-design-mobile/2296)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -164,7 +164,7 @@
     "build-rn": "npm run tsc  && npm run build-rn-ios && npm run build-rn-android",
     "lint-staged": "lint-staged",
     "lint-staged:ts": "npm run tsc && node node_modules/tslint/bin/tslint -c node_modules/antd-tools/lib/tslint.json components/**/*.tsx",
-    "lint-staged:es": "eslint ./.eslintrc.js ./webpack.config.js",
```
Comment (paranoidjk) [inline package.json]: should change to something like `"lint-staged:es": "eslint {tests,site,scripts,components}/**/*.{js,jsx}",` cc @yesmeck  check antd, i guess both configure wrong

Comment (yesmeck) [reply package.json]: https://github.com/ant-design/ant-design/blob/master/package.json#L203-L205

File: .eslintrc.js [hunk 2]
```diff
@@ -1,20 +1,20 @@
-'use strict';
```
Comment (paranoidjk) [inline .eslintrc.js]: 这整个文件看起来没有实质上的改动必要？

**Comment by paranoidjk** [comment]
CI 还是挂的？ 试试 Rebase master.

**Pull request closed without merging**